pilot/cmd/pilot-agent/status/server_test.go

```diff
@@ -217,9 +217,10 @@ my_other_metric{} 0
 			if rec.Code != 200 {
 				t.Fatalf("handleStats() => %v; want 200", rec.Code)
 			}
-			if rec.Body.String() != tt.output {
+			if !strings.Contains(rec.Body.String(), tt.output) {
 				t.Fatalf("handleStats() => %v; want %v", rec.Body.String(), tt.output)
 			}
+			fmt.Println(rec.Body.String())
```

Comment:
We probably need to remove this from merging?

Reply:
done